### PR TITLE
Update polkadotjs and api-augment to latest compatible versions

### PR DESCRIPTION
**Pull request by saraswatpuneet**
TODO:

Pertaining to breaking changes
- [x] Run e2e tests
- [x] Run local + testnet  test

### CODE REVIEW

File: src/blockchain/extrinsic.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export class Extrinsic<T extends ISubmittableResult = ISubmittableResult, C exte
   public payWithCapacity(nonce?: number): Promise<ParsedEventResult> {
     return firstValueFrom(
       this.api.tx.frequencyTxPayment
-        .payWithCapacity(this.extrinsicCall)
+        .payWithCapacity(this.extrinsicCall.toHex())
```
Comment (saraswatpuneet) [inline src/blockchain/extrinsic.ts:95]: I think SubmittableExtrinsic will need a check as this is no longer accepted 

Note: I will test if it works.

Comment (wilwade) [reply src/blockchain/extrinsic.ts:95]: This might not work if our prior test data is based on pre-runtime v110

**Comment by saraswatpuneet** [comment]
@wilwade  e2e tests passes with all latest dependencies

image

**Review by wilwade — APPROVED** [review]
Ship it!

**Pull request merged**